package/MDAnalysis/topology/ITPParser.py

```diff
@@ -495,6 +495,7 @@ def parse(self, include_dir='/usr/local/gromacs/share/gromacs/top/',
         MDAnalysis *Topology* object
         """
 
+        self.atomypes = {}
```

Comment:
```suggestion
        self.atomtypes = {}
```

Typo?

Reply:
I have no idea what happened here, refactor stuff I guess... should be atomtypes